CHANGES.md

```diff
@@ -1,5 +1,25 @@
 # Changes
 
+## edge-20.5.5
+
+This edge release adds refinements to the Linkerd multicluster implementation,
+adds new health checks for the tracing add-on, and addresses an issue in which
+outbound requests from the proxy result in looping behavior.
+
+* CLI
+  * Added `allow` and `link` commands, effectively enabling a cluster to have
```

Comment:
didn't we also rename the `cluster` command to `multicluster`?

Reply:
The `cluster` command wasn't exposed before, IIUC? So we should really call this out as "introduce the multicluster subcommand"

Reply:
👍 Changes describes introduction of the `multicluster` command and its subcommands.